.gitignore

```diff
@@ -16,6 +16,7 @@ __pycache__/
 node_modules/
 artifacts/
 .mypy_cache
+pkg/chain/ethereum/gen/abi/**/*.abi
```

Comment:
We should have it as `*.abi`. With the current version, I have the files from `go/abi` in git diff.